src/researches/getSentenceBeginnings.js

```diff
@@ -51,19 +50,6 @@ const compareFirstWords = function( sentenceBeginnings, sentences ) {
 	return consecutiveFirstWords;
 };
 
-/**
- * Sanitizes the sentence.
- *
- * @param {string} sentence The sentence to sanitize.
- * @returns {string} The sanitized sentence.
- */
-function sanitizeSentence( sentence ) {
-	sentence = stripTags( sentence );
-	sentence = sentence.replace( /^[^A-Za-zА-Яа-я0-9]/, "" );
-
-	return sentence;
-}
-
```

Comment:
I agree that removing `sentence.replace` is justified. I do not see why `stripTags` should not be executed. As you rightly said, `getWords` sanitises words, but it does not apply to the sentence. Since after removing the `sentence.replace` this function is left with just one rule, I would remove the function altogether, but add stripTags as the first rule of `getSentenceBeginning`.